### DISCUSSION TITLE
Columns Keep Resetting?

### DISCUSSION THREAD

**Discussion by emrecio** [keepassxreboot/keepassxc#9556] [Q&A] [answered]
So in macos the columns tend (and I use that term loosely) TEND to stay the way I left them. However on my linux boxes the columns keep getting reset. I don't need the attachment/totp/notes etc. So I am assuming I am doing something wrong? Is it a security setting? Something missing with dbus? IDK. It's just an annoying thing that I've put up with and decided to write to see if there's a solution.

Point/question: is there a way I can keep my column settings even after I reboot my linux box?

**Comment by droidmonkey** [comment, marked as answer]
You are likely mounting you ~/.cache to a ram mount. This will cause the local settings file to be reset on log out / restart. You can choose to NOT do that or use one of our environment variables or command line switches to move where your local settings file is located.

https://keepassxc.org/docs/KeePassXC_UserGuide#_command_line_options

Specifically `KPXC_CONFIG_LOCAL`
